### ISSUE TITLE
Remootio hostname instead of IP address?

### ISSUE THREAD

**Issue by ccorcos** [ronniepettersson/homebridge-remootio#12]
I have two remootios (two different gates) and I'd like to be able to set configuration using a local hostname like "maingate.local" and "backgate.local" instead of an IP address that can change. Any ideas how to do this?

Thanks!

**Comment by ronniepettersson** [comment]
@ccorcos, my Remootio devices seem to be hardcoded with the hostnames in this format: remootio_<serialnumber>.local 

I found the device serial number on the Bluetooth configuration page in the Remootio iPhone app. An alternate way is to use the Discovery app on your iOS device and look for the _remootio._tcp service. 

With the hostname in hand, I changed the IP address to the hostname in the homebridge configuration, and it worked for me.

**Comment by ccorcos** [comment]
Hmm. When I run `arp -a`, I'm seeing two remootios with the same remootio.local address.

```
❯❯❯ arp -a
chets-mbp.attlocal.net (192.168.1.177) at f4:d4:88:79:9a:d7 on en0 ifscope permanent [ethernet]
remootio.attlocal.net (192.168.1.200) at 8c:4b:14:78:ca:98 on en0 ifscope [ethernet]
remootio.attlocal.net (192.168.1.222) at e8:31:cd:66:a2:40 on en0 ifscope [ethernet]
```

The IP address is reachable
```
❯❯❯ ssh root@192.168.1.222
ssh: connect to host 192.168.1.222 port 22: Connection refused
```

But I can't get the local address to work...

```
❯❯❯ ssh root@remootio.local
❯❯❯ ssh root@remootio8c461478ca98ecrsqjlj.local
❯❯❯ ssh root@remootio_8c461478ca98ecrsqjlj.local
```

**Comment by ronniepettersson** [comment]
I think you have the wrong serial number. the first number 46 should be 4b:

remootio.attlocal.net (192.168.1.200) at 8c:**4b**:14:78:ca:98 on en0 ifscope [ethernet]

❯❯❯ ssh root@remootio_8c**46**1478ca98ecrsqjlj.local

**Comment by ccorcos** [comment]
Ah you're right! The image-to-text on my iPhone parsed this bonkers font wrong!

Thanks for the help.

**Comment by ccorcos** [comment]
```
❯❯❯ ssh root@remootio_8c4b1478ca98ecrsqjlj.local
ssh: connect to host remootio_8c4b1478ca98ecrsqjlj.local port 22: Connection refused
```

The config seems unhappy with a local address though.

<img width="720" alt="image" src="https://user-images.githubusercontent.com/1794527/206584648-1078ac02-c3a4-4508-b3dc-8fc77c4ea69e.png">

**Comment by ronniepettersson** [comment]
That seems like a bug/enhancement to the form.

If you go directly into the config editor, you can bypass the form check and make it work.

**Comment by ronniepettersson** [comment]
Fixed in release 1.3.2

**Comment by ccorcos** [comment]
Thank you so much for being so responsive! 😄 💯
